web/ensign-user-ui/public/humans.txt

```diff
@@ -0,0 +1,14 @@
+/* TEAM */
```

Comment:
This will be updated with the team's info in the future. I will have to check about other's who should be added like Pete for example.